common/test/acceptance/tests/studio/test_import_export.py

```diff
@@ -183,6 +184,36 @@ def test_upload(self):
         self.import_page.upload_tarball(self.tarball_name)
         self.import_page.wait_for_upload()
 
+    def test_import_timestamp_value(self):
+        """
+        Scenario: I perform a course / library import
+            On import success, the page displays a UTC timestamp
+        """
+        self.import_page.upload_tarball(self.tarball_name)
+        self.import_page.wait_for_upload()
+
+        utc_now = datetime.utcnow()
+        import_date, import_time = self.import_page.timestamp
+
+        self.assertTrue(self.import_page.is_timestamp_visible())
+        self.assertEqual(utc_now.strftime('%m/%d/%Y'), import_date)
+        self.assertEqual(utc_now.strftime('%H:%M'), import_time)
+
+    def test_import_timestamp_visibility(self):
+        """
+        Scenario: I perform a course / library import
+            On import success, the page displays a timestamp previously not visible
+            And if I refresh the page, the timestamp is still displayed
+        """
```

Comment:
Initially I made it a single test, then I split into two because I was worried I was testing too much stuff in just one test :smile: 

Done, now it's one test only